remoteappmanager/application.py

```diff
@@ -23,7 +23,7 @@ class Application(web.Application, LoggingMixin):
 
     user = Instance(User, allow_none=True)
 
-    db = Instance(orm.Database, allow_none=True)
+    db = Instance(ABCAccounting, allow_none=True)
```

Comment:
I agree that the name isn't great.  `accounting`?

> what happens when we decide to handle more tasks than just accounting in the database?

It seems that the implementation does not stop you from adding more methods to ABCAccounting.  The question is just whether the name `ABCAccounting` would still be appropriate for those future tasks I don't know what.  Is this only a naming issue then?  Or there is more to it?